cmd/admin-handlers.go

```diff
@@ -2636,6 +2636,61 @@ func checkConnection(endpointStr string, timeout time.Duration) error {
 	return nil
 }
 
+type clusterInfo struct {
+	DeploymentID string `json:"deployement_id"`
+	ClusterName  string `json:"cluster_name"`
+	UsedCapacity uint64 `json:"used_capacity"`
+	Info         struct {
+		MinioVersion    string `json:"minio_version"`
+		PoolsCount      int    `json:"pools_count"`
+		ServersCount    int    `json:"servers_count"`
+		DrivesCount     int    `json:"drives_count"`
+		BucketsCount    uint64 `json:"buckets_count"`
+		ObjectsCount    uint64 `json:"objects_count"`
+		TotalDriveSpace uint64 `json:"total_drive_space"`
+		UsedDriveSpace  uint64 `json:"used_drive_space"`
+	} `json:"info"`
+}
+
+func getClusterInfoWithTimeout(ctx context.Context, objectAPI ObjectLayer, timeout time.Duration) (ci clusterInfo, err error) {
```

Comment:
Is this to make it cheaper than whatever is included in regular `mc admin info` response? I would honestly just include that.

Reply:
> Is this to make it cheaper than whatever is included in regular mc admin info response? I would honestly just include that.

This is the requirement for now (I think this will be consumed by another external API later, so we kind of need to have a stable JSON structure)